back-end/db.js

```diff
@@ -1,5 +1,5 @@
 const mysql = require('mysql2');
-
+require('dotenv').config()
```

Comment:
No need for this to be here, in the server you require once before you require the DB file, you will only need to remove it here